browser/ui/commander/commander_service.cc

```diff
@@ -56,7 +56,7 @@ CommanderService::CommanderService(Profile* profile)
   command_sources_.push_back(std::make_unique<BraveSimpleCommandSource>());
   command_sources_.push_back(std::make_unique<BookmarkCommandSource>());
   command_sources_.push_back(std::make_unique<WindowCommandSource>());
-  command_sources_.push_back(std::make_unique<TabCommandSource>());
+  command_sources_.push_back(std::make_unique<BraveTabCommandSource>());
```

Comment:
IMO, it would be great if we can do dedup here when adding new command source instead of doing it from each source?

Reply:
I could probably deduplicate based on the text of the command, but we don't have anything else to go off and it feels a bit brittle.

FWIW, I think it's unlikely that Chromium is going to change this under us (except to remove it), and in that case I think it's probably better that we review it before adding it anyway. We'll need our own version of this class anyway, when we add translations.